src/modules/dome.wren

```diff
@@ -45,6 +45,10 @@ class Version {
 
 class Process {
   foreign static f_exit(n)
+  foreign static args
+
+  static arguments { args[2..-1] }
```

Comment:
This actually just crashes if you haven't got enough arguments.

There's really no way to know how many DOME-specific arguments are going to be used in a commandline call.

Reply:
The a little improvement should be done for checking `f_args.count` before deleting the first element